frontend/lib/api/form/pollingstation/pollingStationUtils.ts

```diff
@@ -393,6 +393,13 @@ export function getClientState(formState: FormState, acceptWarnings: boolean, co
   return clientState;
 }
 
+export function getDataEntryProgress(formState: FormState) {
```

Comment:
Testing with election `3` has the progress bar jump from 38% to 63% after submitting the Differences page, i.e. when you get to the page of "Lijst 1".